<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pages_mini_list</class>
 <widget class="QWidget" name="pages_mini_list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>39</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="fr_list_1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>39</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="horizontalSpacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="fr_block_2_page_1">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_15">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushB_sys_page_1">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
        <widget class="QPushButton" name="pushB_add_sys_page_1">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>9</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, #dadbde, stop: 1 #f6f7fa);
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #dae2ec, stop: 0.4 #d6dde7,
 	stop: 0.5 #bfd7fb, stop: 1.0 #dae3ef);

 	border: 1px solid #cdc2bf;
 	border-radius: 8px;
		
	image: url(:/menu/page/add_list_0.png);
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
	stop:0 rgba(255, 241, 241, 200), 
	stop:0.181818 rgba(255, 255, 255, 135),
	stop:	0.329545 rgba(200, 173, 173, 135), 
	stop:0.681818 rgba(207, 181, 181, 129), 
	stop:0.829545 rgba(255, 255, 255, 129), 
	stop:1 rgba(255, 239, 239, 192));

	image: url(:/menu/page/add_list_1.png);
 }

QPushButton:checked
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #f39556, stop: 0.4 #f7be98,
	stop: 0.5 #f0ccb4, stop: 1.0 #eda16e);

	image: url(:/menu/page/del_page_0.png);
}
 
QPushButton:pressed 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #afc1db, stop: 0.4 #a1b3cf,
	stop: 0.5 #639bee, stop: 1.0 #b3caec);

	image: url(:/menu/page/add_list_1.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushB_del_sys_page_1">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>9</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, #dadbde, stop: 1 #f6f7fa);
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #dae2ec, stop: 0.4 #d6dde7,
 	stop: 0.5 #bfd7fb, stop: 1.0 #dae3ef);

 	border: 1px solid #cdc2bf;
 	border-radius: 8px;
	
	image: url(:/menu/page/del_page_0.png);
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
	stop:0 rgba(255, 241, 241, 200), 
	stop:0.181818 rgba(255, 255, 255, 135),
	stop:	0.329545 rgba(200, 173, 173, 135), 
	stop:0.681818 rgba(207, 181, 181, 129), 
	stop:0.829545 rgba(255, 255, 255, 129), 
	stop:1 rgba(255, 239, 239, 192));

	image: url(:/menu/page/del_page_1.png);
 }

QPushButton:checked
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #f39556, stop: 0.4 #f7be98,
	stop: 0.5 #f0ccb4, stop: 1.0 #eda16e);

	image: url(:/menu/page/del_page_0.png);
}
 
QPushButton:pressed 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #afc1db, stop: 0.4 #a1b3cf,
	stop: 0.5 #639bee, stop: 1.0 #b3caec);

	image: url(:/menu/page/del_page_1.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>pushB_sys_page_1</zorder>
        <zorder>pushB_del_sys_page_1</zorder>
        <zorder>pushB_add_sys_page_1</zorder>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="fr_block_2_page_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_14">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushB_sys_page_2">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
        <widget class="QPushButton" name="pushB_add_sys_page_2">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>9</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, #dadbde, stop: 1 #f6f7fa);
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #dae2ec, stop: 0.4 #d6dde7,
 	stop: 0.5 #bfd7fb, stop: 1.0 #dae3ef);

 	border: 1px solid #cdc2bf;
 	border-radius: 8px;
		
	image: url(:/menu/page/add_list_0.png);
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
	stop:0 rgba(255, 241, 241, 200), 
	stop:0.181818 rgba(255, 255, 255, 135),
	stop:	0.329545 rgba(200, 173, 173, 135), 
	stop:0.681818 rgba(207, 181, 181, 129), 
	stop:0.829545 rgba(255, 255, 255, 129), 
	stop:1 rgba(255, 239, 239, 192));

	image: url(:/menu/page/add_list_1.png);
 }

QPushButton:checked
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #f39556, stop: 0.4 #f7be98,
	stop: 0.5 #f0ccb4, stop: 1.0 #eda16e);

	image: url(:/menu/page/del_page_0.png);
}
 
QPushButton:pressed 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #afc1db, stop: 0.4 #a1b3cf,
	stop: 0.5 #639bee, stop: 1.0 #b3caec);

	image: url(:/menu/page/add_list_1.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushB_del_sys_page_2">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>9</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, #dadbde, stop: 1 #f6f7fa);
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #dae2ec, stop: 0.4 #d6dde7,
 	stop: 0.5 #bfd7fb, stop: 1.0 #dae3ef);

 	border: 1px solid #cdc2bf;
 	border-radius: 8px;
	
	image: url(:/menu/page/del_page_0.png);
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
	stop:0 rgba(255, 241, 241, 200), 
	stop:0.181818 rgba(255, 255, 255, 135),
	stop:	0.329545 rgba(200, 173, 173, 135), 
	stop:0.681818 rgba(207, 181, 181, 129), 
	stop:0.829545 rgba(255, 255, 255, 129), 
	stop:1 rgba(255, 239, 239, 192));

	image: url(:/menu/page/del_page_1.png);
 }

QPushButton:checked
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #f39556, stop: 0.4 #f7be98,
	stop: 0.5 #f0ccb4, stop: 1.0 #eda16e);

	image: url(:/menu/page/del_page_0.png);
}
 
QPushButton:pressed 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #afc1db, stop: 0.4 #a1b3cf,
	stop: 0.5 #639bee, stop: 1.0 #b3caec);

	image: url(:/menu/page/del_page_1.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>pushB_sys_page_2</zorder>
        <zorder>pushB_add_sys_page_2</zorder>
        <zorder>pushB_del_sys_page_2</zorder>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="fr_block_2_page_3">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_16">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushB_sys_page_3">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>3</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
        <widget class="QPushButton" name="pushB_add_sys_page_3">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>9</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, #dadbde, stop: 1 #f6f7fa);
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #dae2ec, stop: 0.4 #d6dde7,
 	stop: 0.5 #bfd7fb, stop: 1.0 #dae3ef);

 	border: 1px solid #cdc2bf;
 	border-radius: 8px;
		
	image: url(:/menu/page/add_list_0.png);
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
	stop:0 rgba(255, 241, 241, 200), 
	stop:0.181818 rgba(255, 255, 255, 135),
	stop:	0.329545 rgba(200, 173, 173, 135), 
	stop:0.681818 rgba(207, 181, 181, 129), 
	stop:0.829545 rgba(255, 255, 255, 129), 
	stop:1 rgba(255, 239, 239, 192));

	image: url(:/menu/page/add_list_1.png);
 }

QPushButton:checked
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #f39556, stop: 0.4 #f7be98,
	stop: 0.5 #f0ccb4, stop: 1.0 #eda16e);

	image: url(:/menu/page/del_page_0.png);
}
 
QPushButton:pressed 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #afc1db, stop: 0.4 #a1b3cf,
	stop: 0.5 #639bee, stop: 1.0 #b3caec);

	image: url(:/menu/page/add_list_1.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushB_del_sys_page_3">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>9</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, #dadbde, stop: 1 #f6f7fa);
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #dae2ec, stop: 0.4 #d6dde7,
 	stop: 0.5 #bfd7fb, stop: 1.0 #dae3ef);

 	border: 1px solid #cdc2bf;
 	border-radius: 8px;
	
	image: url(:/menu/page/del_page_0.png);
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
	stop:0 rgba(255, 241, 241, 200), 
	stop:0.181818 rgba(255, 255, 255, 135),
	stop:	0.329545 rgba(200, 173, 173, 135), 
	stop:0.681818 rgba(207, 181, 181, 129), 
	stop:0.829545 rgba(255, 255, 255, 129), 
	stop:1 rgba(255, 239, 239, 192));

	image: url(:/menu/page/del_page_1.png);
 }

QPushButton:checked
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #f39556, stop: 0.4 #f7be98,
	stop: 0.5 #f0ccb4, stop: 1.0 #eda16e);

	image: url(:/menu/page/del_page_0.png);
}
 
QPushButton:pressed 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #afc1db, stop: 0.4 #a1b3cf,
	stop: 0.5 #639bee, stop: 1.0 #b3caec);

	image: url(:/menu/page/del_page_1.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>pushB_sys_page_3</zorder>
        <zorder>pushB_add_sys_page_3</zorder>
        <zorder>pushB_del_sys_page_3</zorder>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QFrame" name="fr_block_2_page_4">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_17">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushB_sys_page_4">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>4</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
        <widget class="QPushButton" name="pushB_add_sys_page_4">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>9</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, #dadbde, stop: 1 #f6f7fa);
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #dae2ec, stop: 0.4 #d6dde7,
 	stop: 0.5 #bfd7fb, stop: 1.0 #dae3ef);

 	border: 1px solid #cdc2bf;
 	border-radius: 8px;
		
	image: url(:/menu/page/add_list_0.png);
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
	stop:0 rgba(255, 241, 241, 200), 
	stop:0.181818 rgba(255, 255, 255, 135),
	stop:	0.329545 rgba(200, 173, 173, 135), 
	stop:0.681818 rgba(207, 181, 181, 129), 
	stop:0.829545 rgba(255, 255, 255, 129), 
	stop:1 rgba(255, 239, 239, 192));

	image: url(:/menu/page/add_list_1.png);
 }

QPushButton:checked
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #f39556, stop: 0.4 #f7be98,
	stop: 0.5 #f0ccb4, stop: 1.0 #eda16e);

	image: url(:/menu/page/del_page_0.png);
}
 
QPushButton:pressed 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #afc1db, stop: 0.4 #a1b3cf,
	stop: 0.5 #639bee, stop: 1.0 #b3caec);

	image: url(:/menu/page/add_list_1.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushB_del_sys_page_4">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>9</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, #dadbde, stop: 1 #f6f7fa);
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #dae2ec, stop: 0.4 #d6dde7,
 	stop: 0.5 #bfd7fb, stop: 1.0 #dae3ef);

 	border: 1px solid #cdc2bf;
 	border-radius: 8px;
	
	image: url(:/menu/page/del_page_0.png);
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
	stop:0 rgba(255, 241, 241, 200), 
	stop:0.181818 rgba(255, 255, 255, 135),
	stop:	0.329545 rgba(200, 173, 173, 135), 
	stop:0.681818 rgba(207, 181, 181, 129), 
	stop:0.829545 rgba(255, 255, 255, 129), 
	stop:1 rgba(255, 239, 239, 192));

	image: url(:/menu/page/del_page_1.png);
 }

QPushButton:checked
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #f39556, stop: 0.4 #f7be98,
	stop: 0.5 #f0ccb4, stop: 1.0 #eda16e);

	image: url(:/menu/page/del_page_0.png);
}
 
QPushButton:pressed 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
 	stop: 0 #afc1db, stop: 0.4 #a1b3cf,
	stop: 0.5 #639bee, stop: 1.0 #b3caec);

	image: url(:/menu/page/del_page_1.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>pushB_sys_page_4</zorder>
        <zorder>pushB_add_sys_page_4</zorder>
        <zorder>pushB_del_sys_page_4</zorder>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>14</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
